chromium/browser_type_test.go

```diff
@@ -150,3 +153,86 @@ func TestBrowserTypePrepareFlags(t *testing.T) {
 		})
 	}
 }
+
+func TestExecutablePath(t *testing.T) {
+	t.Parallel()
+
+	// we pick a random file name to look for in our tests
+	// this doesn't matter as long as it's in the paths we look for
+	// in ExecutablePath function.
+	const chromiumExecutable = "google-chrome"
+
+	tests := map[string]struct {
+		path        string                            // user provided path
+		lookPath    func(file string) (string, error) // determines if a file exists
+		userProfile env.LookupFunc
```

Comment:
I'm not sure `userProfile` is a good name for this, why not `lookup` or something similar?

Reply:
I get it now, happy for you to ignore this.